src/components/common/layout/HeadingText.tsx

```diff
@@ -75,7 +92,7 @@ const HeadingText: React.FC<HeadingTextProps> = ({
               fontWeight="500"
               lineHeight="16px"
               color="#4D4639"
```

Comment:
Add color in the theme and then use every were

Reply:
@sonaliTekdi and I will do it after API integration